packages/forms/src/documents/types.ts

```diff
@@ -7,6 +7,14 @@ export type DocumentFieldValue =
       maxLength?: number;
       required: boolean;
     }
+  | {
+      type: 'Attachment';
```

Comment:
If I understand correctly, this isn't wired up? If the attachment document-related stuff is a placeholder for the future, IMO we should leave it out for now, unless we're certain that it's correct. @jimmoffet could you weigh in?

Reply:
I think we need attachments for generation but not extraction. We'll need some way to store serialized attachment objects and then process them into supplemental pages at generation time (at which point they become part of the canonical pdf type), but I don't foresee attachments coming in via the form-builder's "start by uploading a pdf" extraction process

Reply:
Removed as discussed yesterday. The types here are used for the document mapping, which appears to be used for the extract methods. We can add it back later if needed, but it should be safe to remove for now.